src/js/actions/deviceActions.js

```diff
@@ -908,3 +933,47 @@ export const setDeviceTags = (deviceId, tags) => dispatch =>
       .catch(err => commonErrorHandler(err, `There was an error setting tags for device ${deviceId}.`, dispatch, 'Please check your connection.'))
       .then(() => Promise.resolve(dispatch({ type: DeviceConstants.RECEIVE_DEVICE, device: { ...device, tags } })));
   });
+
+export const getDeviceTwin = (deviceId, integrationProvider) => (dispatch, getState) =>
+  GeneralApi.get(`${apiBase}${DeviceConstants.EXTERNAL_PROVIDER[integrationProvider].managementUrl}/device/${deviceId}/twin`)
```

Comment:
This was my fault, the path name should be `/devices/{id}/twin` the docs were wrong.
```suggestion
  GeneralApi.get(`${apiBase}${DeviceConstants.EXTERNAL_PROVIDER[integrationProvider].managementUrl}/devices/${deviceId}/twin`)
```

Reply:
should now point to the right endpoint 👌